include/wx/private/webrequest_curl.h

```diff
@@ -87,6 +97,10 @@ class wxWebRequestCURL : public wxWebRequestImpl
     wxObjectDataPtr<wxWebResponseCURL> m_response;
     wxObjectDataPtr<wxWebAuthChallengeCURL> m_authChallenge;
     wxFileOffset m_bytesSent;
+    bool m_cancelPending;
+    bool m_hasInternalFail;
+    wxString m_internalFailMsg;
+    wxFFile m_devNull;
```

Comment:
It looks a bit wasteful (file descriptors are limited in the system) to open this file for each transfer, so I'd move this to the session or maybe even make it a global.

Reply:
These have all been removed since a different way of failing transfers was made in 4c8f9ff